src/components/RightMenu.tsx

```diff
@@ -61,7 +67,7 @@ export const RightMenu: React.FC = () => {
         </StyledA>
       </LinksBlock>
       <Select
-        value={FLUENCE_CLIENT_NETWORK}
+        value={CHAIN_ID == 9999999 ? 'mainnet' : FLUENCE_CLIENT_NETWORK}
```

Comment:
Maybe `9999999` can be a constant declared somewhere?

Reply:
Sure!